src/App.tsx

```diff
@@ -14,11 +16,125 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const numbers: number[] = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+const App: React.FC = () => {
+  const selectedGoods = [...goodsFromServer];
+  const numbersSelect = [...numbers];
+
+  const [isVisible, setIsVisible] = useState(false);
+  const [numberValue, setNumberValue] = useState(0);
+  const [sortBy, setSortBy] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
```

Comment:
you can remove these variables and reverse and sort the array into separate functions